fastmri/data/subsample.py

```diff
@@ -186,9 +204,13 @@ def calculate_center_mask(
 
     def choose_acceleration(self):
         """Choose acceleration based on class parameters."""
-        choice = self.rng.randint(0, high=len(self.accelerations))
-        center_fraction = self.center_fractions[choice]
-        acceleration = self.accelerations[choice]
+        choice_cf = self.rng.randint(0, high=len(self.center_fractions))
```

Comment:
My preferred approach would be to do something like:

```
if sync_center_fractions_accelerations:
    return self.rng.choice(zip(self.center_fractions, self.acceleration))
else:
    return self.rng.choice(self.center_fractions), self.rng.choice(self.acceleration)
```

Reply:
Elegant :)